### PR TITLE
Add HTML option for Sweet alert

### PR DESCRIPTION
**Pull request by KhaledSMQ**
Example:
Alert()->error('Sorry , You are no\'t part of any school. For more information <a href="...../school/invitation">Click Here</a>', 'No school to review.')->html()->persistent('Close');

Thank you in advance

### CODE REVIEW

**Comment by KhaledSMQ** [comment]
ping @uxweb

File: src/SweetAlert/SweetAlertNotifier.php [hunk 1]
```diff
@@ -223,4 +223,20 @@ private function hasTitle()
...
     }
+    
+    /**
+     * Make Message HTML view.
+     * 
+     * @param bool|true $html
+     *
+     * @return \UxWeb\SweetAlert\SweetAlertNotifier $this
+     */
+    public function html($html = true)
```
Comment (uxweb) [inline src/SweetAlert/SweetAlertNotifier.php]: @KhaledSMQ Is there a case when using this method and pass a false?

Comment (KhaledSMQ) [inline src/SweetAlert/SweetAlertNotifier.php]: Hi @uxweb 
Thank for replay 

sorry i think i missed it while testing on my local machine. am going to update

Comment (KhaledSMQ) [inline src/SweetAlert/SweetAlertNotifier.php]: Thank you @uxweb sorry it was mistake i have added while testing on my local machine and forget to remove it. i have commit update :+1:

Comment (KhaledSMQ) [inline src/SweetAlert/SweetAlertNotifier.php]: ping @uxweb

Thank you i have removed unnecessary $html tag and i don't see it's important .

sorry look like i lost between the requests. 

Thank you.

**Comment by uxweb** [comment]
@KhaledSMQ Thank you!! :) :+1:

**Pull request merged**